components/css/buckyless/header.less

```diff
@@ -112,4 +112,28 @@ portal-header {
   .header__bell {
     display: inline-block;
   }
+
+  .header__guest-login-btn {
+    font-weight: 600;
+
+    &:hover,
+    &:focus {
+      color: @white;
+      text-decoration: underline;
+      background-color: rgba(255, 255, 255, 0.11) !important;
+    }
+
+  }
```

Comment:
[linter flags this](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679112260#L754): Unexpected empty line before closing brace.